Source/Core/DolphinWX/Frame.h

```diff
@@ -172,6 +176,11 @@ class CFrame : public CRenderFrame
     ADD_PANE_CENTER
   };
 
+#ifdef __APPLE__
+  IOPMAssertionID m_disable_screensaver_assertion_id;
+  IOReturn m_disable_screensaver_assertion_creation_success;
```

Comment:
Please shorten these names a bit, they are needlessly long. And could just as easily be called  `m_disable_screensaver_id` and `m_disable_screensaver_success` without any loss of information.